base.lua

```diff
@@ -152,7 +152,7 @@ function Qless.jobs(now, state, ...)
     elseif state == 'depends' then
       return queue.depends.peek(now, offset, count)
     elseif state == 'recurring' then
-      return queue.recurring.peek(math.huge, offset, count)
+      return queue.recurring.peek('+inf', offset, count)
```

Comment:
It's odd to me that this would change any behavior. This seems like maybe it is a platform specific bug on Windows rather than a bug in qless's code as `math.huge` is the correct way to make a `+inf` literal value according to the documentation. Here's multiple references stating as much:
* http://lua-users.org/lists/lua-l/2008-12/msg00259.html
* http://www.lua.org/manual/5.3/manual.html#pdf-math.huge
* http://lua-users.org/wiki/MathLibraryTutorial